src/components/Training/Courses.js

```diff
@@ -24,15 +24,19 @@ const Courses = ({ categories, toggleModal }) => {
           <Col width={[1, 1, 1, 1, 1 / 2]} key={cat.id}>
             <Padding bottom={{ smallPhone: 3, tablet: 5 }}>
               <Padding bottom={1}>
-                <Image image={cat.logo} width="60px" />
+                <Image image={cat.logo} width="60px" id={cat.slug} />
                 <SectionTitle>{cat.name}</SectionTitle>
               </Padding>
               <Flex column alignStart>
                 {cat.courses.map(course => (
                   <StyledLink
                     style={{ cursor: 'pointer' }}
-                    onClick={() => toggleModal({ ...cat, ...course })}
+                    to={`/training/${cat.slug}/${course.slug}`}
                     key={course.id}
+                    state={{
+                      modal: true
+                    }}
+                    data-testid={`modal/${cat.slug}/${course.slug}`}
```

Comment:
I'd change slightly the `data-testid` attribute so it's more meaningful, like: `${course.slug}-link`but that's just a matter of taste

Reply:
I reckon that [the `data-testid` convention](https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change/) is quite a nice one, for distinguishing quickly between props which are passed purely for testing purposes vs having any affect on the functionality or styling?

Reply:
sure, I'm not saying to remove the prop, I'm just saying that the prop value could be different (probably wasn't explaining myself right=

Reply:
Oh sorry! Gotcha! :+1: